gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcExceptionBidiStreamingCallable.java

```diff
@@ -54,12 +55,11 @@
   }
 
   @Override
-  public ApiStreamObserver<RequestT> bidiStreamingCall(
-      ApiStreamObserver<ResponseT> responseObserver, ApiCallContext context) {
-
-    GrpcExceptionTranslatingStreamObserver<ResponseT> innerObserver =
-        new GrpcExceptionTranslatingStreamObserver<>(responseObserver, exceptionFactory);
-
-    return innerCallable.bidiStreamingCall(innerObserver, context);
+  public ClientStream<RequestT> call(
+      final ResponseObserver<ResponseT> responseObserver, ApiCallContext context) {
+    return innerCallable.call(
+        new GrpcExceptionServerStreamingCallable.ExceptionResponseObserver(
```

Comment:
Could we extract out ExceptionResponseObserver? Generally I would expect an inner class to be used in conjunction with its outer class, but this is independent of its outer class.

Reply:
Done